util/format/format.go

```diff
@@ -43,6 +43,13 @@ type indentFormatter struct {
 	state       int
 }
 
+var replace = map[rune]string{
+	'\'': "''",
+	'\a': "\\a",
```

Comment:
I'm not quite understand this table.
Why '0' need to be take special?
What does '\a' mean here and is there test cover it?

Reply:
'\a' is useless, I forget to remove it.
'0' is special, because when it used in the table's comment, the process is not the same as others, such as field's comment.